pilot/pkg/proxy/envoy/v2/debounced_channel.go

```diff
@@ -0,0 +1,144 @@
+// Copyright 2019 Istio Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package v2
+
+import (
+	"sync"
+	"time"
+)
+
+// DebouncedEvent is used as entry type, which is enqueued in DebouncedChannel
+type DebouncedEvent = bool
```

Comment:
This is the first time i see this style. Could you use the more common style?

Reply:
Using `type DebouncedEvent bool` would require type conversion from `bool` to `DebouncedEvent` everywhere, the `DebounceChannel` is used.

Reply:
ic